### PR TITLE
[bitnami/mongodb-sharded] Add metrics exporter for shard arbiter

### PR DESCRIPTION
**Pull request by ncmans**
**Description of the change**

Adds metric exporter for shard arbiter pods.

**Benefits**

This seems to have been missed as the Prometheus scraping annotations are there when `metrics.enabled = true` but not the exporter container. This causes Prometheus to mark the arbiter pods as down (ie `up == 0`) and in turn, in our case, sets off an alarm.

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/mongodb-sharded/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mongodb-sharded
-version: 1.2.0
+version: 1.2.1
```
Comment (marcosbc) [inline bitnami/mongodb-sharded/Chart.yaml:3]: Could you increase this as a minor version (i.e. 1.3.0)? Even though it is fixing an issue, it is adding new functionality to the existing chart.

**Comment by marcosbc** [comment]
Thanks for your contribution! Could you check my comment? Once fixed we can launch our CI system and merge the change after tests succeed.

File: bitnami/mongodb-sharded/Chart.yaml (hunk 1 above)
Comment (ncmans) [reply bitnami/mongodb-sharded/Chart.yaml]: sure — done

**Comment by marcosbc** [comment]
@ncmans We launched our CI jobs but it failed: In order to update the image tags to point to latest releases, our CI/CD needs to modify you PR (`bitnami-bot` will update `values.yaml` and `values-production.yaml` for you). To do so, it's necessary you allow "edits from maintainers". Find more information in the link below: 
- https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Could you grant permissions (at least temporarily) for our CI system to be able to make the changes and close this PR? Thanks!

**Comment by ncmans** [comment]
I can't seem to find this option on this PR. Will have to find out why.

**Comment by marcosbc** [comment]
@ncmans Could you try to add bitnami-bot as a collaborator and to allow edits from it?

Screen Shot 2020-04-28 at 18 00 32

**Comment by ncmans** [comment]
@marcosbc done

**Comment by marcosbc** [comment]
@ncmans Unfortunately it looks like bitnami-bot is still unable to modify that repo. I guess it could be related to it belonging to an org.

In any case, the tests in our CI went fine so for now I will merge this PR to unblock this.

Note that, because of that, the chart 1.3.1 will be the released version (1.3.0 will not be published), meaning you should expect to see your changes there.

**Comment by ncmans** [comment]
> @ncmans Unfortunately it looks like bitnami-bot is still unable to modify that repo. I guess it could be related to it belonging to an org.
Ye I figured as much. Thanks for getting it through.

**Comment by carrodher** [comment]
As expected, those changes were already released in the Helm repository using the `1.3.1`, you can find them by running:
```
$ helm repo update
$ ▶ helm search repo bitnami/mongodb-sharded
NAME                   	CHART VERSION	APP VERSION	DESCRIPTION
bitnami/mongodb-sharded	1.3.1        	4.2.6      	NoSQL document-oriented database that stores JS...
```

Please don't hesitate to create an issue if something doesn't work as expected.

**Pull request merged**